### ISSUE TITLE
Bootstrap Grid

### ISSUE THREAD

**Issue by NunoBentes** [oyejorge/less.php#166]
Hi All

Another thing i found... i´m not sure what´s the problem, but i´m trying to compile the Bootstrap´s grid.
I downloaded all the mixins and Variables Only for the Grid, but it´s not working. I used Koala (Desktop Less compiler) to compile the code, and it works as expected. Is this something bug related to oyejorge/less.php ???? I rode in oyejorge documentation and it says that this lib it´s boostrap cpmpatible...

Does anyone having the same problem?

Many thanks in advance

**Comment by chrizz1001** [comment]
Yes I am also unable to compile. The compiler throws me with *lessphp fatal error: .make-grid-columns is undefined in /path/to/less/grid.less*
